ocfweb/docs/views/servers.py

```diff
@@ -13,9 +13,9 @@
 PUPPET_CERT_DIR = '/etc/ocfweb/puppet-certs'
 
 
-class Host(namedtuple('Host', ['hostname', 'type', 'description', 'children'])):
+class Host(namedtuple('Host', ['hostname', 'type', 'description', 'children', 'ssh_keys'])):
```

Comment:
Instead of having `ssh_keys` be a dictionary, we should have two properties, `rsa_key` and `dsa_key` which are just strings. This means we don't have to have four functions to get these keys.

Reply:
![servers_with_keys](https://user-images.githubusercontent.com/30168760/54453761-37f1f580-4715-11e9-81d2-4ee859fca87b.png)